templates/show_params.mako

```diff
@@ -205,6 +205,30 @@
     ${ render_msg( 'One or more of your original parameters may no longer be valid or displayed properly.', status='warning' ) }
 %endif
 
+%if job and job.dependencies:
```

Comment:
@martenson
Should we maybe render this information only if the user is admin? I think the gist I took from https://github.com/galaxyproject/galaxy/pull/2913 is that users are already plenty confused by the differences in versions between galaxy wrappers and tool version. I guess we may be complicating things even more.